sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1541,8 +1541,8 @@ object SQLConf {
       .internal()
       .doc("Prune nested fields from a logical relation's output which are unnecessary in " +
         "satisfying a query. This optimization allows columnar file format readers to avoid " +
-        "reading unnecessary nested column data. Currently Parquet is the only data source that " +
-        "implements this optimization.")
+        "reading unnecessary nested column data. Currently Parquet and ORC v1 are the " +
+        "data sources that implement this optimization.")
       .booleanConf
       .createWithDefault(false)
```

Comment:
@dbtsai @dongjoon-hyun We turned on this flag by default in the upcoming 3.0 because Apple has tried this in the production in the last few months. I am wondering if that statement also includes ORC nested schema pruning?

Reply:
We mainly use it and test it with Parquet.